toolboxes/core/cpu/math/hoNDArray_linalg.cpp

```diff
@@ -590,5 +590,51 @@ template void linFit(const hoNDArray<double>& x, const hoNDArray<double>& y, dou
 template void linFit(const hoNDArray< std::complex<float> >& x, const hoNDArray< std::complex<float> >& y, std::complex<float>& a, std::complex<float>& b);
 template void linFit(const hoNDArray< std::complex<double> >& x, const hoNDArray< std::complex<double> >& y, std::complex<double>& a, std::complex<double>& b);
 
+template <typename T>
+void polyFit2(const hoNDArray<T>& x, const hoNDArray<T>& y, T& a, T& b, T& c)
```

Comment:
We should probably have some kind of unit test for this?

Reply:
test added.